Deployment/default.ps1

```diff
@@ -60,3 +60,15 @@ Task CreateRap -Depends RestoreBuildTools, PackageBuild {
 	}
 	Copy-Item $scriptrunner_rap -Destination "$Packages\$Version\Milyli.ScriptRunner-$Version.rap" -Force | Out-Null
 }
+
+Task CreateNugetPkg -Depends PackageBuild {
+	if(-Not (Test-Path "$Packages\$Version")) {
+		New-Item "$Packages\$Version" -ItemType Directory
+	}
+	else {
+		Remove-Item (Join-Path "$Packages\$Version" "Milyli.ScriptRunner.Services*")
```

Comment:
Does this work because of the `*`? I thought powershell would yell if it tried to delete non-existent files?

Reply:
It hasn't caused any issues when testing this condition.